test/jinahub/encoder/image/test_clip.py

```diff
@@ -0,0 +1,68 @@
+from jina import Flow, Document, DocumentArray
+
+from jinahub.encoder.image.clip import CLIPImageEncoder
+import numpy as np
+
+
+def test_clip_no_batch():
+    def validate_callback(resp):
+        assert 1 == len(DocumentArray(resp.data.docs).get_attributes('embedding'))
+
+    f = Flow().add(uses=CLIPImageEncoder)
+    with f:
+        f.post(on='/test', inputs=[Document(blob=np.ones((3, 224, 224)))], on_done=validate_callback)
+
+
+def test_clip_batch():
```

Comment:
maybe u can add pytest parametrization and  test batch size larger than actual request size, and maybe also combine with traversal path?